Apps/MineOS/docker-compose.yml

```diff
@@ -0,0 +1,162 @@
+      # creates it, it will do so with UID. If an account with this UID
+      # already exists, account creation will be aborted.
+      USER_UID: 1000
+
+      # If the account is being created, it will use either the password
+      # set below, or a random, 20 character password will generated for
+      # you and displayed in the console log. After first launch, you
+      # should check the log.
+      USER_PASSWORD: random_see_log
+
+    ports:
+      # HTTPS port for web interface. If you change the SERVER_PORT
+      # variable, also change this mapping.
+      - "8443:8443/tcp"
+
+      # Minecraft server ports. 25565 is the default, but is you plan
+      # on running multiple servers at the same time, each will need
+      # a unique port unless you are using bungiecord.
+      # Expand this range as needed.
+      - "25565-25570:25565-25570/tcp"
+      - "25565-25570:25565-25570/udp"
+
+    x-casaos:
+      envs:
+        - container: TZ
+          description:
+            en_us: TimeZone
+      ports:
+        - container: "8443"
+          description:
+            en_us: WebUI HTTP Port
+          protocol: tcp
+
+        - container: "25565"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25566"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25567"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25568"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25569"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25570"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25565"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25566"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25567"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25568"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25569"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25570"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+      volumes:
+        - container: /var/games/minecraft
+          description:
+            en_us: MineOS directory.
+x-casaos:
+  architectures:
+    - amd64
+    - arm
+    - arm64
+  main: mineos
+  author: Chàra Rose
+  category: Gaming
```

Comment:
Please change this to an existing category from https://github.com/IceWhaleTech/CasaOS-AppStore/blob/main/category-list.json

Most likely it should be `Games`